nuxeo-core-binarymanager-jclouds/src/main/java/org/nuxeo/ecm/blob/jclouds/JCloudsBinaryManager.java

```diff
@@ -109,6 +113,8 @@ public void initialize(String blobProviderId, Map<String, String> properties) th
             throw new RuntimeException("Missing conf: " + BLOBSTORE_MAP_NAME_KEY);
         }
 
+        endpoint = Framework.getProperty(BLOBSTORE_ENDPOINT_KEY);
```

Comment:
If would be nice if you could also convert all these `Framework.getProperty` to use the same property pattern than recent binarymanagers, which allows configuration through the generic descriptor.